src/encode/pattern/mod.rs

```diff
@@ -619,29 +649,37 @@ impl FormattedChunk {
 /// An `Encode`r configured via a format string.
 #[derive(Derivative)]
 #[derivative(Debug)]
-#[derive(Clone, Eq, PartialEq, Hash)]
+#[derive(Clone, Eq, PartialEq)]
```

Comment:
Why remove Hash derive?

Reply:
Cargo was returning an error w/ `Hash` derived. So i removed it and it complied and `cargo test` fine (at least w/ rustc 1.60). 
```
   --> src/encode/pattern/mod.rs:657:5
    |
652 | #[derive(Clone, Eq, PartialEq, Hash)]
    |                                ---- in this derive macro expansion
...
657 |     color_map: HashMap<Level, Option<Color>>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Hash` is not implemented for `HashMap<Level, std::option::Option<Color>>`
    |
    = note: this error originates in the derive macro `Hash` (in Nightly builds, run with -Z macro-backtrace for more info)
 ```

Reply:
I think we'll need to keep it derived for semver reasons